client/src/use/useTextLayer.js

```diff
@@ -35,8 +36,15 @@ export default function useTextLayer({
 
   const textStyle = computed(() => {
     const _selectedTrackId = selectedTrackId.value;
+    const _editingTrackId = editingTrackId.value;
     return {
       color: (data) => {
+        if (_editingTrackId !== null) {
+          if (_editingTrackId !== data.detection.track) {
+            return '#777777'; // color for other detections when editing
```

Comment:
Since both this and `useAnnotationLayer` use this value, it might be good to put it into a constant, which through it's name would also help describes it's function.